{"name": "test_search_success[1-2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "self = <test_a.TestSearch object at 0x00000000040A9DD8>, a = 1, b = 2\n\n    @allure.story(\"搜索成功\")\n    @pytest.mark.parametrize(['a', 'b'], yaml.safe_load(open(\"./data.yaml\")))\n    def test_search_success(self,a,b):\n        # print(\"这是搜索：测试用例，搜索成功\")\n>       assert a == b\nE       assert 1 == 2\n\ntest_a.py:13: AssertionError"}, "parameters": [{"name": "a", "value": "1"}, {"name": "b", "value": "2"}], "start": 1606879639950, "stop": 1606879639950, "uuid": "5f455a3d-1364-4554-a739-540304a4cfe2", "historyId": "ac631ae86304ed1103163d0cc284b090", "testCaseId": "a914917c6a7304d72ac1d949e1329365", "fullName": "test_a.TestSearch#test_search_success", "labels": [{"name": "feature", "value": "搜索模块"}, {"name": "story", "value": "搜索成功"}, {"name": "suite", "value": "test_a"}, {"name": "subSuite", "value": "TestSearch"}, {"name": "host", "value": "LIUHONG-L2"}, {"name": "thread", "value": "17308-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_a"}]}